t/27-consoles-s3270.t

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/perl
+
+use Data::Dumper;
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use Test::Mock::Time;
+use Test::MockModule;
+use Test::MockObject;
+use Test::Output qw(combined_like stderr_like);
+use Test::Warnings qw(:all :report_warnings);
+use POSIX qw(waitpid _exit);
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use consoles::console;
+use backend::ipmi;
+use consoles::VNC;
+use consoles::s3270;
+use cv;
+
+cv::init;
+require tinycv;
```

Comment:
are you sure you need those imports?

Reply:
I have validated them, they are not required. I will remove these imports